app/views/devise/registrations/edit.html.slim

```diff
@@ -27,44 +27,45 @@
             .actions
               = f.submit('Update', class: 'btn btn-primary', disabled: true)
 
-    #add_application_token_form.collapse
-      = form_for :application_token, url: application_tokens_path, remote: true, html: {id: 'new-application-token-form', class: 'form-horizontal', role: 'form'} do |f|
-          .form-group
-            = f.label :application, {class: 'control-label col-md-2'}
-            .col-md-7
-              = f.text_field(:application, class: 'form-control', required: true, placeholder: "Name")
-          .form-group
-            .col-md-offset-2.col-md-7
-              = f.submit('Create', class: 'btn btn-primary')
+    .app-token-wrapper
```

Comment:
The idea is to have an element that contains both the link and the form. I know I could just add classes/ids to the elements and access from the global scope but restricting the component to access only the `this.$el` give us more control of what's going on and also a good practice.

Ideally, panel and form could be separated components and when user clicked on the link an event is triggered and the component that contains both panel and form will tell the form to show itself. What do you think? 

But yeah, wrapping them in a component to restrict dom element scope would still be needed.